### PR TITLE
Es6 modules

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
# Awesome books: with ES6
## In this project:
- ES6 syntax.
- Use ES6 Modules.
- Use npm as a software package system.
- Use Luxon to update the date and time in the UI.
- Create a new directory and initialize it with git.
- create a package.json file using npm (Node Package manager).
- Create a modules directory.
- Display the current date below the navigation bar.
- Add Linters for HTML/CSS & JS.
- Use GitHub flow.
- Use best practices for HTML & CSS & JS.

### CODE REVIEW

File: modules/functions.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      removeBtnsEl.forEach((item) => {
+        item.addEventListener('click', (e) => {
+          bookList.splice(e.target.id, 1);
+          localStorage.setItem('bookList', JSON.stringify(bookList));
+          this.renderBooks();
+        });
+      });
+    };
+    removeBook();
+  }
```
Comment (awais-amjed) [inline modules/functions.js]: Kindly replace these class functions with arrow-style syntax following the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#project-requirements) 🤗

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @MostafaShahat2021 , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/functions.js (hunk 1 above)
Comment (MostafaShahat2021) [reply modules/functions.js]: Hi, @awais-amjed Thanks a lot for your efforts in reviewing my code.✌️😀

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @MostafaShahat2021,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by MostafaShahat2021** [comment]
Hi, @Omar-Muhamad Thanks a lot for your efforts in reviewing my code ✌️.

**Pull request merged**